crates/test-utils/src/transaction.rs

```diff
@@ -95,7 +95,7 @@ pub async fn publish_package_with_wallet(
     let transaction = {
         let data = client
             .transaction_builder()
-            .publish(sender, all_module_bytes, None, GAS_BUDGET)
+            .publish(sender, all_module_bytes, vec![], None, GAS_BUDGET)
```

Comment:
Will need to find a way to get the right dependencies in here -- I think probably the best thing is to have `publish_package_with_wallet` expect a `CompiledPackage` rather than `all_module_bytes`, and ask it what its dependencies are.